runtime/vm/exceptiondescribe.c

```diff
@@ -319,7 +319,7 @@ iterateStackTrace(J9VMThread * vmThread, j9object_t* exception, callback_func_t
 					}
 					if (inlineDepth == 0) {
```

Comment:
Can you set `methodPC = UDATA_MAX;` here or before this if statement so the else clauses aren't needed?

Reply:
There are still a few references of `methodPC` expecting a valid value:
Line 346 
`romClass = findROMClassFromPC(vmThread, methodPC, &classLoader);`, 
Line 370 
`if ((methodPC >= (UDATA)possibleMethod) && (methodPC < (UDATA)J9_BYTECODE_END_FROM_ROM_METHOD(possibleMethod))) {`
Line 381 
`romMethod = findROMMethodInROMClass(vmThread, romClass, methodPC);`

It doesn't seem there is a good place to set `methodPC = UDATA_MAX;` beforehand and save later duplicates.